<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_threadInside">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>内部创建线程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_threadOutside">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>外部创建线程</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_st">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ed">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
